### ISSUE TITLE
Leia compile error

### ISSUE THREAD

**Issue by samnazarko** [palinek/pvr.sledovanitv.cz#43]
Hi,

When building this add-on for Kodi Leia a few hours ago, I'm getting:

```
/mnt/package/mediacenter-osmc/src/xbmc-8e967df9218279618bcbfa8a898d8f80f7b4e449/cmake/addons/build/pvr.sledovanitv.cz/src/Data.cpp: In member function 'bool sledovanitvcz::Data::LoadEPGJob()':
/mnt/package/mediacenter-osmc/src/xbmc-8e967df9218279618bcbfa8a898d8f80f7b4e449/cmake/addons/build/pvr.sledovanitv.cz/src/Data.cpp:186:68: error: invalid conversion from 'long unsigned int*' to 'const time_t* {aka const long int*}' [-fpermissive]
       LoadEPG(start - (start % 86400) - DiffBetweenUtcAndLocalTime(&start), false);
```

It looks like you've made some changes here in the last hour, so I will check if they are now resolved.

**Comment by palinek** [comment]
Yes, this should be fixed in e77be597109c8143ce3052f1f376494ff17625bc

Please, confirm...

**Comment by samnazarko** [comment]
Will do once Jenkins is finished

On 13 Mar 2020, at 19:18, palinek <notifications@github.com> wrote:

﻿

Yes, this should be fixed in e77be59<https://github.com/palinek/pvr.sledovanitv.cz/commit/e77be597109c8143ce3052f1f376494ff17625bc>

Please, confirm...

**Comment by samnazarko** [comment]
It’s fixed
